cmd/maya-apiserver/spc-watcher/lease_test.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2017 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package spc
+
+import (
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	openebsFakeClientset "github.com/openebs/maya/pkg/client/generated/clientset/internalclientset/fake"
+
+	"testing"
+	"github.com/golang/glog"
+)
+
+// SpcCreator will create face spc objects
+func (focs *clientSet) SpcCreator(poolName string, SpcLeaseKeyPresent bool, SpcLeaseKeyValue string) (claim *apis.StoragePoolClaim) {
+	var spcObject *apis.StoragePoolClaim
+	if SpcLeaseKeyPresent {
+		spcObject = &apis.StoragePoolClaim{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: poolName,
+				Annotations: map[string]string{
+					"openebs.io/spc-lease": SpcLeaseKeyValue,
+				},
+			},
+		}
+	} else {
+		spcObject = &apis.StoragePoolClaim{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: poolName,
+			},
+		}
+	}
+	spcGot, err := focs.oecs.OpenebsV1alpha1().StoragePoolClaims().Create(spcObject)
+	if err != nil {
+		glog.Error(err)
+	}
+	return spcGot
+}
+func TestGetLease(t *testing.T) {
+	// Get a fake openebs client set
+	focs := &clientSet{
+		oecs: openebsFakeClientset.NewSimpleClientset(),
+	}
+	// Make a map of string(key) to struct(value).
+	// Key of map describes test case behaviour.
+	// Value of map is the test object.
+	tests := map[string]struct {
+		// fakestoragepoolclaim holds the fake storagepoolcalim object in test cases.
+		fakestoragepoolclaim *apis.StoragePoolClaim
+		// expectedResult holds the expected result for the test case under run.
+		expectedResult string
+	}{
+		// TestCase#1
+		"SPC#1": {
```

Comment:
Little description like. `lease already acquired`, `lease not acquired by any pod` etc would be good

Reply:
That will give a value. I will add it.

Reply:
fixed